data_prepared.jsonl

```diff
@@ -119,3 +119,7 @@
 {"prompt":"ten thousand ->\n\n###\n\n","completion":" 1万, ichi man, 1 man\n"}
 {"prompt":"miracle ->\n\n###\n\n","completion":" 奇跡, きせき, kiseki\n"}
 {"prompt":"the creature ->\n\n###\n\n","completion":" \n"}
+//My changes
+{"prompt":"Water ->\n\n###\n\n","completion":" \n"}
+{"prompt":"Water ->\n\n###\n\n","completion":" \n"}
+{"prompt":"casa ->\n\n###\n\n","completion":"家, いえ, ie\n"}
```

Comment:
Hello thank you for your contribution but this is an english to japanese translator.